lib/pipeline.js

```diff
@@ -929,7 +929,7 @@ class PipelineModel extends BaseModel {
      */
     async getFirstAdmin() {
         const newAdmins = this.admins;
-
+        const handleStatuses = [401, 404];
```

Comment:
can you define this `const` outside of this function ? Also call this perhaps `SCM_NO_ACCESS_STATUSES`...

Reply:
Fixed.